core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/CarbonConstants.java

```diff
@@ -428,4 +430,31 @@ private CarbonConstants() {
     // parameter to indicate whether HTTP access to Admin Console is enabled
     public static final String ENABLE_HTTP_ADMIN_CONSOLE = "EnableHTTPAdminConsole";
 
+    /**
+     * Constants for Logging events.
+     */
+    public static class LogEventConstants {
+
+        // Common keys for both audit/diagnostic logs.
+        public static final String LOG_ID = "id";
+        public static final String RECORDED_AT = "recordedAt";
+        public static final String CLIENT_COMPONENT = "clientComponent";
+        public static final String CORRELATION_ID = "correlationId";
```

Comment:
it was decided to use "requestId" instead of "correlationId"

Reply:
resolved in e61201e